src/services/creategeojsonbloodhound.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @fileoverview Provides a function that creates a Bloodhound engine
+ * expecting GeoJSON responses from the search web service, and creating
+ * `ol.Feature` objects as suggestions.
+ *
+ * Example:
+ *
+ * var bloodhound = ngeoCreateGeoJSONBloodhound(
+ *   'http://example.com/fulltextsearch?query=%QUERY',
+ *   ol.proj.get('EPSG:3857'));
+ * bloodhound.initialize();
+ */
+
+goog.provide('ngeo.CreateGeoJSONBloodhound');
+
+goog.require('ngeo');
+goog.require('ol.format.GeoJSON');
+
+
+/**
+ * @typedef {function(string, ol.proj.Projection=,
+ *     ol.proj.Projection=):Bloodhound}
+ */
+ngeo.CreateGeoJSONBloodhound;
+
+
+/**
+ * @param {string} url URL pattern.
+ * @param {ol.proj.Projection=} opt_featureProjection Feature projection.
+ * @param {ol.proj.Projection=} opt_dataProjection Data projection.
+ * @return {Bloodhound} The Bloodhound object.
+ */
+ngeo.createGeoJSONBloodhound = function(url, opt_featureProjection,
+    opt_dataProjection) {
+  var geojsonFormat = new ol.format.GeoJSON();
+  return new Bloodhound(/** @type {BloodhoundOptions} */ ({
+    remote: {
+      url: url,
+      ajax: {
+        dataType: 'jsonp'
+      },
```

Comment:
Thanks. I changed it, and added an example in the fileoverview showing how to pass a Bloodhound options object.